action.yml

```diff
@@ -0,0 +1,30 @@
+name: pysa
+description: "Run pysa on the repository and check for vulnerabilities"
+
+inputs:
+  build-type:
+    description: Pysa build to use. Possible values are stable and nightly. Default is stable.
+    required: false
+    default: "stable"
+
+runs:
+  using: "composite"
+  steps:
+    - run: if [ ${{inputs.build-type}} = "nighty" ]; then pip install pyre-check-nightly; else pip install pyre-check; fi
+      shell: bash
+    - run: if [ ! -f .pyre_configuration ]; then printf ".\n" | pyre init; fi
+      shell: bash
+    - run: pyre infer
```

Comment:
Before running `pyre infer`, we should also make sure we install the repo's dependencies the action is running on with `pip install -r requirements.txt`

[Step 2 in Initial configuration in the quickstart guide](https://pyre-check.org/docs/pysa-quickstart/#initial-configuration) provides context for why we need to do this

Reply:
Okay.